getting_started/supported_languages.Rmd

```diff
@@ -0,0 +1,24 @@
+---
+title: "Supported languages"
+parent: Getting started
+nav_order: 2
+---
+
+# Supported languages
```

Comment:
Hier kan je eventueel Scala, JavaScript en binnenkort C# ook al bij zetten, maar vermelden dat er nog geen documentatie voor is.

Reply:
Scala, JS en C# toegevoegd met als notitie dat documentatie WIP is.